blocks/medium-promo-block/features/medium-promo/default.test.jsx

```diff
@@ -90,12 +90,37 @@ describe('the medium promo feature', () => {
     const noImgConfig = {
       itemContentConfig: { contentService: 'ans-item', contentConfiguration: {} },
       showHeadline: true,
+      headlinePosition: 'below',
       showImage: false,
     };
     const wrapper = mount(<MediumPromo customFields={noImgConfig} />);
     expect(wrapper.find('.md-promo-image')).toHaveLength(0);
   });
 
+  // it('headline div should have class .headline-above when headline position is above', () => {
```

Comment:
are these deliberately commented out? I thought we wanted these tests

Reply:
Someone-- maybe Rachel? -- commented out the className these were testing for.

Reply:
ok is there some way to fix the tests? the className could be important for css